commit 2cc7abc7f615e3d80985b291472e2cf42e4e60c7
Author: Christoph Cullmann <cullmann@kde.org>
Date:   Sat Jan 13 20:46:11 2018 +0100

    Fix linking regression introduced by
    
    d47f645a7e97526d259a242d31dd829f057e6c9c
    Const'ify. Remove unused variables. Minor optimization

diff --git src/backends/fake/kauth-policy-gen-polkit.cpp src/backends/fake/kauth-policy-gen-polkit.cpp
index d103414..5575620 100644
--- src/backends/fake/kauth-policy-gen-polkit.cpp
+++ src/backends/fake/kauth-policy-gen-polkit.cpp
@@ -38,7 +38,7 @@ const char policy_tag[] = ""
 
 const char dent[] = "   ";
 
-void output(QList<Action> actions, QMap<QString, QString> domain)
+void output(QList<Action> actions, const QMap<QString, QString> &domain)
 {
     Q_UNUSED(domain)
 
diff --git src/backends/mac/kauth-policy-gen-mac.cpp src/backends/mac/kauth-policy-gen-mac.cpp
index e146c46..ccfc0e7 100644
--- src/backends/mac/kauth-policy-gen-mac.cpp
+++ src/backends/mac/kauth-policy-gen-mac.cpp
@@ -26,7 +26,7 @@
 
 using namespace std;
 
-void output(QList<Action> actions, QMap<QString, QString> domain)
+void output(QList<Action> actions, const QMap<QString, QString> &domain)
 {
     AuthorizationRef auth;
     AuthorizationCreate(NULL, kAuthorizationEmptyEnvironment, kAuthorizationFlagDefaults, &auth);
diff --git src/backends/policykit/kauth-policy-gen-polkit.cpp src/backends/policykit/kauth-policy-gen-polkit.cpp
index 4f29cb0..c215275 100644
--- src/backends/policykit/kauth-policy-gen-polkit.cpp
+++ src/backends/policykit/kauth-policy-gen-polkit.cpp
@@ -39,7 +39,7 @@ const char policy_tag[] = ""
 
 const char dent[] = "   ";
 
-void output(QList<Action> actions, QHash<QString, QString> domain)
+void output(QList<Action> actions, const QMap<QString, QString> &domain)
 {
     QTextStream out(stdout);
     out.setCodec("UTF-8");
